assets/chat/js/autocomplete.js

```diff
@@ -197,6 +197,15 @@ class ChatAutoComplete {
         this.selected = -1
         this.results = []
         this.criteria = criteria
+
+        //for emote suffixes
+        if (criteria.orig.includes(' :') && criteria.word.startsWith(':')) {
+            const last_colon = criteria.orig.lastIndexOf(' :')
```

Comment:
Consider naming this `lastColon` to match the naming scheme (camel case) of other variables in the project.